main.js

```diff
@@ -24,9 +26,15 @@ var $nav = createNavbar()
 $nav.classList.add('navbar', 'navbar-inverse')
```

Comment:
Your `$nav` should be ready before `createNavbar` returns. Add the classes in the function that creates it!